test/mocha/12-insert.js

```diff
@@ -95,6 +95,29 @@ describe('insert API', () => {
       error.message.should.equal(
         '"doc.sequence" must be a non-negative integer.');
     });
+  it('should fail to insert a document with an unsafe sequence number',
+    async () => {
+      let record, error = null;
+      try {
+        const actor = actors['alpha@example.com'];
+        const {doc1} = mockData;
+        const doc = {...doc1};
+        doc.id = await helpers.generateRandom();
+        doc.sequence = Number.MAX_SAFE_INTEGER;
```

Comment:
It was agreed that MAX_SAFE_INTEGER is not acceptable, so I changed the description of the test.

Reply:
can you please add comments here and/or in the code that explains the rational behind the choice we made today?  Without all the context that we went over on the call todday, I think it would be difficult for someone to determine why MAX_SAFE_INTEGER is not acceptable.  Something like:

doc.sequence is limited to MAX_SAFE_INTEGER - 1 to avoid unexpected behavior when a client attempts to increment the sequence number.

Reply:
I added the reasoning comment in the _validateDocSequence function in the storage.js file.